groceristar.js

```diff
@@ -1,8 +1,10 @@
 const _            = require('lodash');
 const departments  = require('./data/Departments/departments.json');
-const groceries    = require('./data/Grocery/grocery.json');
+let groceries      = require('./data/Grocery/grocery.json');
```

Comment:
not sure why you change `const` to `let` it's debatable what we should have, but it's at least catching attention when we have other objects from json file, that was described with const